lang/lang-es.js

```diff
@@ -0,0 +1,591 @@
+(function (window) {
+/**
+ * {virtualclass1} {virtualclass2} are elements you passes with getString function eg:-
+ *   virtualclass.lang.getString('operaBrowserIssue', ['opeara', 27]);
+ *   opera and 27 will be replaced over the {virtualclass1} and {virtualclass2} resepectively
+ *   for particular line of language file.
+ * @type type
+ */
+    const message = {
+        ActiveAll: 'Activar todo',
+        addcontext: 'Añadir marcador',
+        Addoption: 'Añadir opción',
+        addnew: 'Añadir nuevo',
+        adminusr: 'Administrador',
+        appPrerequites: 'Validar requisitos del sistema',
+        assign: 'Asignar',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)